### ISSUE TITLE
Edit profile: Some content not marked up as headings

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#40]
URL: https://hcdev.openplus.ca/en/user/203/edit
User Agent: Chrome, Firefox
Language: both

Issue: Page has an H1 on the page but no other headings. Some sections look like they have a headings but those are actually links to collapse the sections. This can confuse the user.

Expected result: Mark up the page using headings. Language settings and Locale settings can both be marked up as H2. If those are not headings then still remove them as links. They don't look like usual collapsible sections (those usually have an arrow pointing right/down).

![image](https://user-images.githubusercontent.com/14280062/128409832-5830621e-6bef-483c-be84-acdbf9ef3976.png)

**Comment by echouin** [comment]
@qanitah This is a Drupal default. They may change this in future Drupal releases. Suggest item be closed.

**Comment by mmiron-HC** [comment]
@echouin We shouldn't be closing issues that are still outstanding. This template is open source. this is a WCAG violation therefore your developer could make the change and submit the fix to Drupal WXT. This would help advance the quality of the template.

**Comment by echouin** [comment]
As noted previously, this is an issue with the WxT of Drupal itself.

**Comment by davidgotlieb** [comment]
OpenPlus has reported that the issue has bene resolved up stream and should now be fixed.

![image](https://user-images.githubusercontent.com/53194802/197041008-aa848814-fa0a-423c-bd3f-7c62fa191e60.png)
